Entity/Document.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+
+namespace EMS\CommonBundle\Entity;
+
+
+class Document
+{
+    /**@var string */
+    private $contentType;
+    /**@var string */
+    private $ouuid;
+    /**@var array */
+    private $source;
+
+    public function __construct(string $contentType, string $ouuid, array $source)
+    {
+        $this->contentType = $contentType;
+        $this->ouuid = $ouuid;
+        $this->source = $source;
+    }
+
+    /**
```

Comment:
I would remove redundant docblocks, all information of this doc is already readable in code.

Reply:
Old habit (auto generated by PHP storm)